src/test/java/racingcar/util/RandomNumberGeneratorTest.java

```diff
@@ -0,0 +1,19 @@
+package racingcar.util;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RandomNumberGeneratorTest {
+    private static final int TESTCASE_NUM = 10000;
+
+    @Test
+    @DisplayName("랜덤 숫자 생성")
+    void generateRandomNumber() {
+        final NumberGenerator randomNumber = new RandomNumberGenerator();
+        for (int i = 0; i < TESTCASE_NUM; i++) {
+            assertThat(randomNumber.generate()).isBetween(0, 9);
+        }
+    }
+}
```

Comment:
```suggestion
}

```

혹시 10000번을 돌린 이유가 있나요?